src/Features/Core/Portable/ImplementInterface/AbstractImplementInterfaceService.CodeAction.cs

```diff
@@ -309,7 +309,7 @@ private string DetermineMemberName(ISymbol member, ArrayBuilder<ISymbol> impleme
                 var generateInvisibleMember = GenerateInvisibleMember(member, memberName);
                 memberName = generateInvisibleMember ? member.Name : memberName;
 
-                var generateAbstractly = !generateInvisibleMember && Abstractly;
+                var generateAbstractly = !member.IsStatic && !generateInvisibleMember && Abstractly;
```

Comment:
possibly explicitly comment that the language doesn't allow static abstract impls of interface methods.

Reply:
Added comment.